brittany.yaml

```diff
@@ -0,0 +1,17 @@
+conf_layout:
+  lconfig_indentPolicy: IndentPolicyLeft
+  lconfig_cols: 80
+  lconfig_indentAmount: 2
+conf_forward:
+  options_ghc:
+  - -XLambdaCase
+  - -XMultiWayIf
+  - -XGADTs
+  - -XPatternGuards
+  - -XViewPatterns
+  - -XTupleSections
```

Comment:
We don't necessarily need to list all the extensions. Some are already natively supported. However I need to test this current config on other repos to update it. 

In a future PR. I plan to extract the extensions from the cabal file of each formatted project using https://github.com/kowainik/extensions